### PR TITLE
New weights for GH

### PR DESCRIPTION
**Pull request by crixal**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3114)

### CODE REVIEW

**Comment by crixal** [comment]
test now~

**Comment by crixal** [comment]
test now!

**Comment by crixal** [comment]
test!

File: src/main/scala/beam/router/r5/CarWeightCalculator.scala [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import org.matsim.core.router.util.TravelTime
+
+import java.util.concurrent.ThreadLocalRandom
+import java.util.concurrent.atomic.AtomicInteger
+
+class CarWeightCalculator(workerParams: R5Parameters, travelTimeNoiseFraction: Double = 0D) {
+  private val networkHelper = workerParams.networkHelper
+  private val transportNetwork = workerParams.transportNetwork
+
+  val maxFreeSpeed: Double = workerParams.networkHelper.allLinks.map(_.getFreespeed).max
```
Comment (JustinPihony) [inline src/main/scala/beam/router/r5/CarWeightCalculator.scala:13]: Why not just use the helper `networkHelper` already extracted on line 10?

**Review by JustinPihony — APPROVED** [review]
Minor nit - otherwise lgtm

File: src/main/scala/beam/router/r5/CarWeightCalculator.scala (hunk 1 above)
Comment (crixal) [reply src/main/scala/beam/router/r5/CarWeightCalculator.scala]: Fixed, thanks!

**Comment by crixal** [comment]
test!

**Pull request merged**